lib/streamlit/web/server/app_static_file_handler.py

```diff
@@ -30,10 +31,14 @@
 MAX_APP_STATIC_FILE_SIZE = 200 * 1024 * 1024  # 200 MB
 # The list of file extensions that we serve with the corresponding Content-Type header.
 # All files with other extensions will be served with Content-Type: text/plain
-SAFE_APP_STATIC_FILE_EXTENSIONS = (".jpg", ".jpeg", ".png", ".gif")
+SAFE_APP_STATIC_FILE_EXTENSIONS = (".jpg", ".jpeg", ".png", ".gif", ".webp")
 
 
 class AppStaticFileHandler(AssetsFileHandler):
+    def initialize(self, path: str, default_filename: Optional[str] = None) -> None:
+        super().initialize(path, default_filename)
+        mimetypes.add_type("image/webp", ".webp")
```

Comment:
We add `mimetypes.add_type("image/webp", ".webp")` here, because `.webp` extension is known on some systems (e.g. MacOS, and auto-discovered automatically), but missing on others (e.g. on Ubuntu Latest)